spec/api-deprecations-spec.js

```diff
@@ -76,6 +76,22 @@ describe('deprecations', () => {
     expect(o).to.have.a.property(oldPropertyName).that.is.equal(value)
   })
 
+  it('deprecates a property of an object', () => {
+    let msg
+    deprecations.setHandler((m) => { msg = m })
+
+    const propertyName = 'itMustGo'
+    let value = 0
+
+    let o = { [propertyName]: value }
+    expect(o).to.have.a.property(propertyName).that.is.a('number')
```

Comment:
I'm not sure if that `expect()` is really needed.
I just create this very object, and basically checks that the syntax is correct )